libs/vertexai/tests/integration_tests/test_chat_models.py

```diff
@@ -140,6 +142,62 @@ def test_multimodal() -> None:
     assert isinstance(output.content, str)
 
 
+multimodal_inputs = [
+    pytest.param(
+        "gs://cloud-samples-data/generative-ai/video/pixel8.mp4",
+        "video/mp4",
+        id="video",
+    ),
+    pytest.param(
+        "gs://cloud-samples-data/generative-ai/audio/pixel.mp3", "audio/mp3", id="audio"
+    ),
+    pytest.param(
+        "gs://cloud-samples-data/generative-ai/image/cricket.jpeg",
+        "image/jpeg",
+        id="image",
+    ),
+]
+
+
+@pytest.mark.release
+@pytest.mark.parametrize("file_uri,mime_type", multimodal_inputs)
+def test_multimodal_media_file_uri(file_uri, mime_type) -> None:
+    llm = ChatVertexAI(model_name="gemini-1.5-pro-preview-0514")
+    media_message = {
+        "type": "media",
+        "file_uri": file_uri,
+        "mime_type": mime_type,
+    }
+    text_message = {
+        "type": "text",
+        "text": "Describe the attached media in 5 words!",
+    }
+    message = HumanMessage(content=[text_message, media_message])
+    output = llm([message])
+    assert isinstance(output.content, str)
+
+
+@pytest.mark.release
+@pytest.mark.parametrize("file_uri,mime_type", multimodal_inputs)
+def test_multimodal_media_inline_base64(file_uri, mime_type) -> None:
+    llm = ChatVertexAI(model_name="gemini-1.5-pro-preview-0514")
+    storage_client = storage.Client()
+    blob = storage.Blob.from_string(file_uri, client=storage_client)
+    media_base64 = base64.b64encode(blob.download_as_bytes()).decode()
```

Comment:
could you include a test that passes a gcs_uri, please?

Reply:
See test_multimodal_media_file_uri.